mkdocs.yml

```diff
@@ -0,0 +1,66 @@
+## Project information
+site_name: Python SDK
+site_description: Kili Python SDK
+site_url: https://kili-technology.github.io/kili-playground/
+
+## Repository
+repo_url: https://github.com/kili-technology/kili-playground
+repo_name: SDK on GitHub
```

Comment:
Kili Python SDK

Reply:
for this one I persist, since we have:
<img width="182" alt="image" src="https://user-images.githubusercontent.com/1255753/165515815-044fd710-6f84-417a-8d8e-d615eaa52f4d.png">

Reply:
yes, I changed it in local. I am just waiting for the URL to be changed to push all the changes